packages/discord.js/src/structures/GuildMember.js

```diff
@@ -320,6 +320,11 @@ class GuildMember extends Base {
    * Edits this member.
    * @param {GuildMemberEditOptions} options The options to provide
    * @returns {Promise<GuildMember>}
+   * @example
+   * // Deafen a guild member
+   * guildMember.edit({ deaf: true })
+   *   .then(member => console.log(`Deafened ${member.nickname}`))
+   *   .catch(console.log);
```

Comment:
Well `VoiceState#setDeaf()` is for that

Reply:
Might want to add an info tag since almost all `GuildMember#edit()` options are already able to be applied with another method, which would be more readable and concise. Or delete the example as a whole

Reply:
i think just delete it

Reply:
Agreed, this is a weirdly specific example for the edit method. It probably doesn't need an example as it's uncommon for this to be called directly.